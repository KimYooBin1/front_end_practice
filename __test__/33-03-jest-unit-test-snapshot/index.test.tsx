
import {render} from '@testing-library/react'
import "@testing-library/jest-dom"
import JestUnitPage from '../../pages/section33/33-03-jest-unit-test-snapshot'

it("기존 사진과 바뀐게 없는지 확인해보자!! - snapshot test",()=>{
	const result = render(<JestUnitPage />)
	//처음 실행시 사진을 찍어서 저장하게 되고 그다음부터 해당 snapshot 과 비교한다
	//yarn test -u 옵션을 통해 snapshot 을 업데이트 할 수 있다.
	expect(result.container).toMatchSnapshot()	
})