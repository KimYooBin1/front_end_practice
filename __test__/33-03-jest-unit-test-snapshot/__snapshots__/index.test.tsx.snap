// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`기존 사진과 바뀐게 없는지 확인해보자!! - snapshot test 1`] = `
<div>
  <div>
    철수는 14살 입니다.
  </div>
  철수의 취미 입력하기:
  <input
    type="text"
  />
  <button>
    철수랑 놀러가기
  </button>
</div>
`;
